### PR TITLE
adds event for git workshop

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

File: content/W2016/W16-event-gitworkshop.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Title: Technical Workshop: Intro to Git and Open-Source
+Date: 2016-03-03 14:00
+Category: Events
+Tags: Blackberry, workshop, git, open-source
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md]: Second B should be capitalized, and are our other workshops missing this tag? (That'd be a second PR)

File: content/W2016/W16-event-gitworkshop.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+Title: Technical Workshop: Intro to Git and Open-Source
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md]: Should this be "Open Source Software"?

File: content/W2016/W16-event-gitworkshop.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Date: 2016-03-03 14:00
+Category: Events
+Tags: Blackberry, workshop, git, open-source
+Slug: git-workshop-w16
+Author: Fatema Boxwala
+Summary: Join us for a fun workshop to learn the basics of Git, Github and
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md]: Needs to be a single line and not wrapped

File: content/W2016/W16-event-gitworkshop.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Date: 2016-03-03 14:00
+Category: Events
+Tags: Blackberry, workshop, git, open-source
+Slug: git-workshop-w16
+Author: Fatema Boxwala
+Summary: Join us for a fun workshop to learn the basics of Git, Github and 
+open-source contributions!
+
+What is Git? Why are people always telling me to make branches? How does one 
+contribute to an open-source project? We’ll be answering all these questions
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md:10]: No - in open source.

File: content/W2016/W16-event-gitworkshop.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+and more at our Intro to Git and Open-Source Workshop! Fatema will (try) to
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md]: Axe the "(try)"

File: content/W2016/W16-event-gitworkshop.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+open-source contributions!
+
+What is Git? Why are people always telling me to make branches? How does one 
+contribute to an open-source project? We’ll be answering all these questions
+and more at our Intro to Git and Open-Source Workshop! Fatema will (try) to 
+explain the basics of Git, and by the end of the workshop all participants will
+have submitted a pull request to a public repository. Upper year mentors will
+be around to answer questions and dinner will be provided!
+
+This workshop is recommended for beginners, you can view the repo we’ll be working
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md]: Comma splice, maybe do "beginners, and you" or "beginners: you"

File: content/W2016/W16-event-gitworkshop.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+on [here](https://github.com/fboxwala/git-workshop-W16). Navigate to the wiki
+tab to view the full curriculum. Space for the workshop is limited so make
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md]: "Wiki tab" should probably be a link.

File: content/W2016/W16-event-gitworkshop.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+our [sponsors page]({filename}/pages/sponsors.md)
+
+Note: This is a [women-only](http://wics.uwaterloo.ca/faq) event.
+
+## Event Details ##
+
++ **Who:** [Women-only](http://wics.uwaterloo.ca/faq)
++ **What:** Git Workshop
++ **Where:** STC 0050
++ **When:** Tuesday March 15, 5:30&ndash;7:30PM
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md:35]: Two hours is pretty short, I should probably take a look at your curriculum for you...

File: content/W2016/W16-event-gitworkshop.md (hunk 1 above)
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md]: Also the "open source" tag [doesn't have a dash](http://wics.uwaterloo.ca/tags/open-source/)

Comment (fboxwala) [inline content/W2016/W16-event-gitworkshop.md]: no I already added the blackberry tag to the ardiuno one

File: content/W2016/W16-event-gitworkshop.md (hunk 8 above)
Comment (fboxwala) [inline content/W2016/W16-event-gitworkshop.md:35]: pls do. I designed it to be pretty short, a one night thing. you can take a look and let me know if you think 2 hours wont be enough.

File: content/W2016/W16-event-gitworkshop.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+What is Git? Why are people always telling me to make branches? How does one 
+contribute to an open-source project? We’ll be answering all these questions
+and more at our Intro to Git and Open Source Software Workshop! Fatema will
+to explain the basics of Git, and by the end of the workshop all participants will
+have submitted a pull request to a public repository. Upper year mentors will
+be around to answer questions and dinner will be provided!
+
+This workshop is recommended for beginners, and you can view the repo we’ll be working
+on [here](https://github.com/fboxwala/git-workshop-W16). Navigate to the [wiki
+tab](https://github.com/fboxwala/git-workshop-W16/wiki) to view the full curriculum. Space for the workshop is limited so make
```
Comment (ehashman) [inline content/W2016/W16-event-gitworkshop.md:18]: This line isn't 80-char justified but w/e.

**Pull request merged**